packages/ui/src/domainModules/agent/components/AgentManager/item/BaseForm/index.js

```diff
@@ -83,5 +100,24 @@ const EnhancedForm = compose(connect(undefined, mapDispatchToProps))(BaseForm)
 
 export default reduxForm({
   enableReinitialize: true,
-  validate: formValidator({ model: 'normalizedAgent' }),
+  shouldError: params => {
+    if (params) {
+      const { props, nextProps } = params
+
+      return (
+        (props && props.registeredFields) !==
+          (nextProps && nextProps.registeredFields) ||
+        (props && props.values) !== (nextProps && nextProps.values)
+      )
+    }
+
+    return false
+  },
+  updateUnregisteredFields: true,
```

Comment:
I don't think you need `updateUnregisteredFields` and `shouldError`. is this just a copy-paste?